scripts/plugins/doc.py

```diff
@@ -1,28 +1,27 @@
-# Loads JSON
-# Provides a function to access the documentation
-from .logging import logger
 import json
 
-fh = open("scripts/plugins/documentation.json")
-jh = json.load(fh)
 
+def get_doc(lookup_key):
+    """Function to access the documented error message by its code"""
+    msg = None
+    with open("scripts/plugins/documentation.json", encoding="utf8") as fp:
```

Comment:
Will this not cause the file to be reread every time get_doc is called?

I would suggest keeping this at the top of the file as the documentation file will be loaded a single time on the first import of the doc file.

Reply:
Updated in the https://github.com/bitovi/bitops/pull/298/commits/7cc75ed5bc637f9b569ba9b46286021eedde74a1 so we do lazy-load it only once.